twake/frontend/src/app/services/login/login.js

```diff
@@ -321,8 +321,8 @@ class Login extends Observable {
         if (!no_reload) {
           Globals.window.location.reload();
         } else {
-          RouterServices.history.push(
-            RouterServices.pathnames.LOGIN + '?' + RouterServices.history.location.search.substr(1),
+          RouterServices.pushTo(
+            `${RouterServices.pathnames.LOGIN}?${RouterServices.history.location.search.substr(1)}`,
```

Comment:
We have to `substr` again and again and this looks like a bad API design

Reply:
It is not necessary here, it should be:
```
`${RouterServices.pathnames.LOGIN}${RouterServices.history.location.search}`
```

Reply:
updated 👍🏻